playbooks/README.md

```diff
@@ -108,10 +108,13 @@ installed and a config file in ```$HOME/.kube/config```.  Also make sure you
 indentified with the cluster using ```oc login```.
 
 Before installing KubeVirt on an existing OpenShift cluster, ensure that SELinux is disabled on all hosts:
+
 ```bash
 $ ansible-playbook -i inventory playbooks/selinux.yml
 ```
 
+Be sure to use the same inventory configuration file you used to deploy the OpenShift cluster.
```

Comment:
assuming i used a very minimal inventory [file](https://gist.github.com/yuvalif/777b9b992cf11a3505e6577eb664357c) to install openshift. what would be missing there?

Reply:
@yuvalif Nothing. I have just tested it, works for me. Please let me know if it doesn't for you.